backend/muster-roll/README.md

```
@@ -1,27 +1,38 @@
+### Service Dependencies
+
+- DIGIT backbone services
+- Idgen
+- Persister
+- Indexer
+- Workflow
+- User
+- Attendance
+- Expense calculator (for billing period lookups and bill-status lock checks)
+
+## Service Details
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)